packages/report-server/src/reportBuilder/transform/functions/mergeRows/mergeRows.ts

```diff
@@ -70,19 +70,22 @@ const groupRows = (rows: Row[], params: MergeRowsParams) => {
       return;
     }
     const groupKey = params.createGroupKey(row);
-    groupsByKey[groupKey] = addRowToGroup(groupsByKey[groupKey] || {}, row);
+    groupsByKey[groupKey] = addRowToGroup(groupsByKey[groupKey], row);
     parser.next();
   });
 
   return { groups: Object.values(groupsByKey), ungroupedRows };
 };
 
-const addRowToGroup = (group: Group, row: Row): Group => {
-  const newGroup = { ...group };
+const addRowToGroup = (group: Group = {}, row: Row): Group => {
   Object.keys(row).forEach((field: string) => {
-    newGroup[field] = [...(newGroup[field] || []), row[field]];
```

Comment:
If I had a nickel for (each time I made this mistake x each time I made this mistake) I would be a very rich crappy developer 😝